.travis.yml

```diff
@@ -26,7 +26,8 @@ matrix:
 cache:
   - pip
 before_install:
-  - pip install --upgrade pip
+  # Pinned to version 19.3.1 due to error while running tests (tox) on Travis
```

Comment:
Nit: is there a better way to enforce a pip version that makes this more configurable (e.g. requirements file?)

Reply:
Can't think of one. This was a suggestion I found in another pr.